packages/experimental/Button/src/ButtonTokens.ts

```diff
@@ -17,6 +17,48 @@ export const defaultButtonTokens: TokenSettings<ButtonTokens, Theme> = (t: Theme
     fluid: {
       width: '100%',
     },
+    fab: {
+      borderRadius: 100, // big number for always rounded corners
+      shadowColor: '#000000',
```

Comment:
nit: you could use the global Shadow tokens here if one fits the same values, but not blocking

Reply:
I took a quick look, and it looks like the shape of the global tokens is different from the props in View. I.E:
```
// globalTokens.shadow.2
"shadow": {
  "2": [
      {
          "x": 0,
          "y": 0,
          "blur": 2,
          "color": "#0000001f"
      },
      {
          "x": 0,
          "y": 1,
          "blur": 2,
          "color": "#00000024"
      }
  ],
  ...
}
```
vs what RN / iOS wants:
```
style: {
  shadowColor: ColorValue,
  shadowOffset: {
    width: number,
    height: number,
  },
  shadowRadius: number,
  shadowOpacity: number,      
}
```

To use the globalTokens, we will probably need some appropriate conversion layer?

Reply:
You should be able to pass in the color as-is, without needing opacity? or is opacity strictly needed? Blur is the same as radius, I think

Reply:
Actually, on a second pass I think you are right. 
shadowColor -> color (first 6 hex values)
shadowOffset -> x / y
blur -> radius
shadowOpacity -> color (last 2 hex values)

Reply:
@lindaweng If you can't get to that before the end of your internship, I'll log a bug on our end to use the global tokens as a followup

Reply:
The global shadow tokens 8 does match the same values, but I would only be able to use it for shadowOffset and shadowRadius. They correspond to the x, y, and blur values in global shadow tokens. The color in global tokens combines the color and opacity whereas RN separates them into shadowColor and shadowOpacity. I'm not sure if it would make things more or less confusing with some props set with global shadow tokens and some not.

Reply:
Obviously I didn't see the other comments before making mine, but I'm just not sure how to separate out the color into 2 different parts

Reply:
I got the hex value splitting working. It's not the prettiest, but it works with the global shadow tokens.

Reply:
You could define this helper above (with the helpful inline comments)
```
const shadowStyleFromGlobalToken = (shadowToken: number) => {
  const shadow = globalTokens.shadow[shadowToken][1] // Grab the second shadow from the global token, as we only support displaying one shadow
  return {
    // iOS Shadow props
    shadowColor: shadow.color.substr(0, 7), // split out the color
    shadowOpacity: Number('0x' + shadow.color.substr(7, 2)) / 255.0, // split out the opacity
    shadowRadius: shadow.blur,
    shadowOffset: {
      width: shadow.x,
      height: shadow.y
    },
    // Android shadow props
    elevation: shadowToken 
  }
}
```

and then do:
```
fab: {
  ...shadowStyleFromGlobalToken(8),
  // rest of props
}

```

Reply:
@rurikoaraki to answer your earlier question, I thought shadowOpacity would not be needed but it doesn't render properly unless you provide it. So we still necessarily need it yes.